owncloudApp/lint.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lint>
+    <issue id="WrongConstant" severity="ignore" />
+    <issue id="MissingTranslation" severity="ignore" />
+    <issue id="ByteOrderMark" severity="ignore" />
```

Comment:
Which criteria do you follow to ignore this lints?

Reply:
Without them, it will fail.
Everyone is welcome to remove this, and solve this, after a merge

Reply:
Please see the post for @davigonz 
![image](https://user-images.githubusercontent.com/3314607/70593931-2de71f80-1bdf-11ea-873b-dadbc1d6493f.png)

Reply:
Ok, thanks, i'll check it in #2750